src/scenes/Search/store.js

```diff
@@ -68,13 +67,14 @@ export default class SearchStore {
   }
 
   async fetch(phrase) {
-    let result = [];
-    if (_.isEmpty(phrase)) return result;
+    if (_.isEmpty(phrase)) return [];
     let oracles = await searchOracles(phrase);
     oracles = _.uniqBy(oracles, 'txid').map((oracle) => new Oracle(oracle, this.app));
-    let topics = await searchTopics(phrase);
+    const topicFilters = [];
+    topicFilters.push({ status: OracleStatus.WITHDRAW });
```

Comment:
inline this with topicFilters: `const topicFilters = [{ status: OracleStatus.WITHDRAW }];`